client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -3,24 +3,37 @@ import Box from '@material-ui/core/Box';
 import { Link } from 'react-router-dom';
 import useStyles from './useStyles';
 import { Typography } from '@material-ui/core';
+import Logo from '../../Images/logo.png';
+import AuthMenu from '../AuthMenu/AuthMenu';
 
 interface Props {
-  linkTo: string;
+  linkTo?: string;
   asideText: string;
-  btnText: string;
+  btnText?: string;
 }
 
-const AuthHeader = ({ linkTo, asideText, btnText }: Props): JSX.Element => {
+const AuthHeader = ({ asideText }: Props): JSX.Element => {
   const classes = useStyles();
 
   return (
-    <Box p={1} className={classes.authHeader}>
-      <Typography className={classes.accAside}>{asideText}</Typography>
-      <Link to={linkTo} className={classes.link}>
-        <Button color="inherit" className={classes.accBtn} variant="contained">
-          {btnText}
-        </Button>
+    <Box className={classes.authHeader}>
+      <Link to="/">
+        <img src={Logo} alt="logo" />
       </Link>
+      <Box p={1} className={classes.authMenus}>
+        <Typography className={classes.accAside}>{asideText}</Typography>
+        <Link to="/login" className={classes.link}>
+          <Button className={`${classes.headerBtn} ${classes.loginBtn}`} color="primary" variant="outlined">
+            LOGIN
```

Comment:
Please avoid using CAPITALCASE as much as possible because some screen readers pick them up as acronyms. You can instead use the CSS class to transform the text to uppercase :)

Ditto throughout!

Reply:
Applied